### PR TITLE
NitinJ_Assignment_8

### PR DESCRIPTION
**Pull request by NitinJOct2020**
Write the same program using the while loop including all operations.

### CODE REVIEW

File: src/nitin/WhileLoop.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
+package nitin;
+
+public class WhileLoop {
+	
+	void evenNumber(int start,int end) {
+		int index = start;
```
Comment (harshvegada) [inline src/nitin/WhileLoop.java:6]: why do we need extra variable here?
you can use start directly as increment variable.

**Review by harshvegada — APPROVED** [review]
Code matches requirement but there some scope of improvement.

Refer this code:
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/142/files

File: src/nitin/WhileLoop.java (hunk 1 above)
Comment (NitinJOct2020) [reply src/nitin/WhileLoop.java:6]: Thank you Harsh for your comment, I will try it based on your suggestion, do i have to submit the assignment again in the same pull request?

Comment (NitinJOct2020) [reply src/nitin/WhileLoop.java:6]: Thank you i have tried again and it worked, i am pushing it as a practice assignment.

**Pull request merged**